File: src/main/kotlin/com/riskrieg/mapeditor/util/MathUtil.java
package com.riskrieg.mapeditor.util;

import java.awt.Point;

public class MathUtil {

  public static double distance(Point one, Point two) {
    int dx = two.x - one.x;
    int dy = two.y - one.y;
    return Math.sqrt((dx * dx) + (dy * dy));
  }

}

File: src/main/kotlin/com/riskrieg/mapeditor/util/MathUtil.kt
package com.riskrieg.mapeditor.util

import java.awt.Point
import kotlin.math.sqrt

object MathUtil {

    fun distance(one: Point, two: Point): Double {
        val dx = two.x - one.x
        val dy = two.y - one.y
        return sqrt((dx * dx + dy * dy).toDouble())
    }

}